ern-container-gen/src/resources/CodePushStarterMiniApp.file

```diff
@@ -0,0 +1,15 @@
+import { Component } from 'react'
+import codePush from 'react-native-code-push'
+
+export default class App extends Component<Props> {
+  render() {
+    return null
+  }
+}
+
+App = codePush({
+  checkFrequency: codePush.CheckFrequency.ON_APP_RESUME,
+  installMode: codePush.InstallMode.ON_NEXT_RESUME,
+  minimumBackgroundDuration: 60 * 2,
+})(App)
+codePush.sync()
```

Comment:
Seems like we also need to pass the install mode here ... per the documentation : https://github.com/Microsoft/react-native-code-push/blob/master/docs/api-js.md#codepushsync